Docs: Update REST API docs for "Run automation for all failing hosts of a policy" endpoint (#13630)
fleetdm/fleet: docs/REST API/rest-api.md

}
```

### Run automation for all failing hosts of a policy

> Triggers [automations](https://fleetdm.com/docs/using-fleet/automations#policy-automations) for *all* hosts with failing policies, regardless of whether a policy was previously failing on a host.

Reviewer: I think this line says something different, if I understand correctly. Should it instead be...

```suggestion
Triggers [automations](https://fleetdm.com/docs/using-fleet/automations#policy-automations) for *all* hosts that have the specified policy, regardless of whether the policy was previously failing on a host.
```
